resources/js/views/admin/BoundingBox.vue

```diff
@@ -226,8 +231,12 @@ export default {
         /**
          *
          */
-        startDrawingBox (e)
+        mousedownSelf (e)
         {
+            // The user has clicked outside all the boxes, so deselect all boxes:
+            for(var b = 0; b < this.boxes.length; b++) this.boxes[b].selected = false;
```

Comment:
Create a function to reuse instead of duplicate code with line 222.

Reply:
done!